src/com/goide/inspections/GoPlaceholderCountInspection.java

```diff
@@ -162,23 +40,18 @@ public void visitCallExpr(@NotNull GoCallExpr o) {
         PsiReference psiReference = o.getExpression().getReference();
         PsiElement resolved = psiReference != null ? psiReference.resolve() : null;
         if (!(resolved instanceof GoFunctionOrMethodDeclaration)) return;
-        int placeholderPosition = getPlaceholderPosition((GoFunctionOrMethodDeclaration)resolved);
-        List<GoExpression> arguments = o.getArgumentList().getExpressionList();
-        if (placeholderPosition < 0 || arguments.size() <= placeholderPosition) return;
-        
-        GoExpression placeholder = arguments.get(placeholderPosition);
-        if (!isStringPlaceholder(placeholder)) {
-          String message = "Value used for formatting text does not appear to be a string";
-          holder.registerProblem(placeholder, message, ProblemHighlightType.GENERIC_ERROR_OR_WARNING);
-          return;
-        }
 
-        int parameterCount = arguments.size() - placeholderPosition - 1;
-        int placeholdersCount = getPlaceholders(placeholder);
-        if (placeholdersCount == -1 || placeholdersCount == parameterCount) return;
+        String functionName = ((GoFunctionOrMethodDeclaration)resolved).getName();
+        if (functionName == null) return;
```

Comment:
toLowerCase return null for null argument, no needed to check function name here, check it later and avoid variable reassignments